### PR TITLE
fix test

### PR DESCRIPTION
**Pull request by dianakocsis**
## Related Issue

Which issue does this pull request resolve?

### CODE REVIEW

**Review by wjmelements — APPROVED** [review]

File: test/libraries/Pool.t.sol [hunk 1]
```diff
@@ -119,9 +119,12 @@ contract PoolTest is Test {
         assertEq(slot0.protocolFee(), 0);
         slot0 = slot0.setProtocolFee(protocolFee);
         assertEq(slot0.protocolFee(), protocolFee);
+        state.slot0 = slot0;
+
+        uint16 pFee = params.zeroForOne ? protocolFee0 : protocolFee1;
```
Comment (hensha256) [inline test/libraries/Pool.t.sol:124]: i dont like `pFee` 😆 can we rename it something more descriptive? Or rename `protocolFee` and call this one `protocolFee`?

Comment (hensha256) [reply test/libraries/Pool.t.sol:124]: swapProtocolFee? finalProtocolFee? expectedProtocolFee?

**Review by hensha256 — APPROVED** [review]

**Review by hensha256 — APPROVED** [review]

File: test/libraries/Pool.t.sol [hunk 2]
```diff
@@ -121,10 +121,10 @@ contract PoolTest is Test {
         assertEq(slot0.protocolFee(), protocolFee);
         state.slot0 = slot0;
 
-        uint16 pFee = params.zeroForOne ? protocolFee0 : protocolFee1;
+        uint16 expectedProtocolFee = params.zeroForOne ? protocolFee0 : protocolFee1;
 
         uint24 _lpFee = params.lpFeeOverride.isOverride() ? params.lpFeeOverride.removeOverrideFlag() : lpFee;
-        uint24 swapFee = protocolFee == 0 ? _lpFee : pFee.calculateSwapFee(_lpFee);
+        uint24 swapFee = protocolFee == 0 ? _lpFee : expectedProtocolFee.calculateSwapFee(_lpFee);
```
Comment (wjmelements) [inline test/libraries/Pool.t.sol:127]: should it be
```
uint24 swapFee = expectedProtocolFee == 0 ? _lpFee : expectedProtocolFee.calculateSwapFee(_lpFee);
```

Comment (wjmelements) [reply test/libraries/Pool.t.sol:127]: based on
```
            swapFee = protocolFee == 0 ? lpFee : uint16(protocolFee).calculateSwapFee(lpFee);
```

Comment (hensha256) [reply test/libraries/Pool.t.sol:127]: mmmm yes i believe so, good catch

**Review by hensha256 — APPROVED** [review]

**Pull request merged**